pkg/nodeconfig/nodeconfig.go

```diff
@@ -42,6 +44,10 @@ const (
 	WorkerLabel = "node-role.kubernetes.io/worker"
 	// PubKeyHashAnnotation corresponds to the public key present on the VM
 	PubKeyHashAnnotation = "windowsmachineconfig.openshift.io/pub-key-hash"
+	// Condition type for Cloud Controller ownership
```

Comment:
Please start with `cloudControllerOwnershipCondition` as per Go standards

Reply:
Done, also renamed it to cloudControllerOwnershipConditionType